specification/authorization/resource-manager/readme.md

```diff
@@ -45,19 +45,13 @@ These settings apply only when `--tag=package-2015-07` is specified on the comma
 
 ``` yaml $(tag) == 'package-2015-07'
 input-file:
-- Microsoft.Authorization/stable/2015-07-01/authorization.json
+- Microsoft.Authorization/stable/2015-07-01/authorization-RoleDefinitionsCalls.json
+- Microsoft.Authorization/stable/2015-07-01/authorization-ProviderOperationsCall.json
```

Comment:
nit: might as well end the filename in *Calls.json like other files, to be consistent.

Reply:
done!